<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTimer</class>
 <widget class="QDialog" name="NewTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New</string>
  </property>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Aftermath</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>112</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="resetRB">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="repeatRB">
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="deleteRB">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>344</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="descriptionTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="nameTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="timeEdit">
      <property name="displayFormat">
       <string>hh.mm.ss</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
